client/windows/CAdvmapInterface.cpp

```diff
@@ -1721,6 +1721,18 @@ void CAdvMapInt::tileHovered(const int3 &mapPos)
 		const CGPathNode * pnode = LOCPLINT->cb->getPathsInfo(h)->getPathInfo(mapPos);
 		assert(pnode);
 
+		if(LOCPLINT->altPressed() && pnode->reachable()) //overwrite status bar text with movement info
+		{
+			const int maxMovementPointsForDestinationLastTurn = pnode->turns > 0 ? h->maxMovePoints(pnode->layer == EPathfindingLayer::LAND) : h->movement;
+			const int movementPointsLastTurnCost = maxMovementPointsForDestinationLastTurn - pnode->moveRemains;
+			const int remainingPointsAfterMove = pnode->turns == 0 ? pnode->moveRemains : 0;
+
+			const std::string costWord = VLC->generaltexth->allTexts[346];
+			const std::string leftWord = VLC->generaltexth->allTexts[200];

```

Comment:
@IvanSavenko , now we should use VLC->generaltexth->translate, right?

Reply:
Yeah, good idea probably.
```suggestion
			const std::string costWord = VLC->generaltexth->translate("core.genrltxt.346");
			const std::string leftWord = VLC->generaltexth->translate("core.genrltxt.200");

```
For H3 files, format is "core.<filename>.<line>".
You can dump all loaded texts via "convert txt" command